tools/build-version.go

```diff
@@ -38,23 +38,12 @@ func getTag(match ...string) (string, *semver.PRVersion) {
 }
 
 func main() {
-	tags, err := exec.Command("git", "tag").Output()
-	if err != nil {
+	if err := exec.Command("git", "tag").Run(); err != nil {
```

Comment:
Do we still need to run `git tag`? IIUC it was just to check if we need to fetch or not?